Sources/MeiliSearch/Model/KeyParams.swift

```diff
@@ -5,17 +5,42 @@ import Foundation
  */
 public struct KeyParams: Codable, Equatable {
   public let description: String
```

Comment:
I think description is [optional as well](https://github.com/meilisearch/meilisearch-swift/pull/312#pullrequestreview-1060092053)